doc/manual/43-backup-commands.en.md

```diff
@@ -198,6 +198,16 @@ command.
 
 The recovery command has several options that modify the command behavior.
 
+> **IMPORTANT:**
+> When recovering with `--no-get-wal` - whether you recover with 
```

Comment:
@andremagui please reword... you are repeating `--no-get-wal` twice in the same sentence (which is avoidable) also is not really clear what you mean in this whole paragraph 😅

Reply:
I relocate the note to a better section related to get-wal! Also reworded a bit!